moralization/spacy_model_manager.py

```diff
@@ -240,7 +244,9 @@ def _check_model_is_trained_before_it_can_be(self, action: str = "used"):
         if not self._best_model_path.is_dir():
             raise RuntimeError(f"Model must be trained before it can be {action}.")
 
-    def _get_data_manager_docbin_files(self, data_manager: DataManager) -> List[Path]:
+    def _get_data_manager_docbin_files(
+        self, data_manager: DataManager, check_data_integrity=True
```

Comment:
Since you've provided a defalt value for `check_data_integrity` in the user-facing `train` method, no need to repeat this default in this internal function
```suggestion
        self, data_manager: DataManager, check_data_integrity: bool
```